course/meetings/cs-francois-pitt-2017-03-14.md

```diff
@@ -0,0 +1,64 @@
+---
+contact: Francois Pitt
+department: Computer Science
+uoftcoders: Madeleine, Luke, Joel
+date: 2017-03-14
+---
+
+# Action items
+- Joel: Send Francois email with blurb
+- Madeleine: Check with masters supervisor in bio stream?
```

Comment:
We can leave this here as a reminder - I'll wait until we hear back from Francois because it sounds like my committee member will be in the email blast anyway.